pkg/apis/wksprovider/controller/manifests/yaml/04_controller.yaml

```diff
@@ -42,7 +42,7 @@ spec:
         operator: Exists
       containers:
       - name: controller
-        image: weaveworks/cluster-api-existinginfra-controller:v0.2.4
+        image: weaveworks/cluster-api-existinginfra-controller:latest
```

Comment:
I don't think it's safe to use `latest` here. That would mean that any `wk` binary someone used after CAPEI got updated would end up pulling the new manifest (which could easily be inconsistent w/ the binary in some way) :thinking:

Reply:
@jrryjcksn  You're right. I'll revert this.